proprietary/Components/src/denhaag/card-calendar.tokens.json

```diff
@@ -0,0 +1,164 @@
+{
+  "denhaag": {
+    "card-calendar": {
+      "wrapper": {
+        "display": { "value": "grid" },
+        "grid-template-columns": { "value": "repeat(12, 1fr)" },
+        "xs": {
+          "column-gap": { "value": "0" },
+          "row-gap": { "value": "55px" },
+          "grid-column": { "value": "span 12" }
+        },
+        "sm": {
+          "gap": { "value": "16px" },
+          "grid-column": { "value": "span 6" }
+        },
+        "md": {
+          "gap": { "value": "26px" },
+          "grid-column": { "value": "span 4" }
+        },
+        "lg": {
+          "gap": { "value": "31px" },
+          "grid-column": { "value": "span 3" }
+        },
+        "xl": {
+          "gap": { "value": "40px" },
+          "first": {
+            "grid-column-start": { "value": "3" },
+            "grid-column-end": { "value": "5" }
+          },
+          "second": {
+            "grid-column-start": { "value": "5" },
+            "grid-column-end": { "value": "7" }
+          },
+          "third": {
+            "grid-column-start": { "value": "7" },
+            "grid-column-end": { "value": "9" }
+          },
+          "fourth": {
+            "grid-column-start": { "value": "9" },
+            "grid-column-end": { "value": "11" }
+          }
+        }
+      },
+      "item": {
+        "position": { "value": "relative" },
+        "padding-inline": { "value": "8px" },
+        "padding-block": { "value": "8px" },
+        "border": { "value": "1px solid" },
+        "border-color": { "value": "{denhaag.color.grey.2}" },
+        "border-radius": { "value": "8px" },
+        "font-family": { "value": "{denhaag.typography.sans-serif.font-family}" },
+        "height": { "value": "fit-content" },
+        "underline": {
+          "position": { "value": "absolute" },
+          "content": { "value": "''" },
+          "left": { "value": "15px" },
+          "right": { "value": "15px" },
+          "bottom": { "value": "-4px" },
+          "height": { "value": "1px" },
+          "background-color": { "value": "{denhaag.color.grey.2}" }
+        },
+        "hover": {
+          "box-shadow": { "value": "-1px -4px 15px 0px rgba(0,0,0,0.1)" },
+          "cursor": { "value": "pointer" }
+        },
+        "focus": {
+          "border": { "value": "2px dashed" },
+          "border-color": { "value": "{denhaag.color.ocher.5}" }
+        },
+        "header": {
+          "position": { "value": "relative" },
+          "background-color": { "value": "{denhaag.color.grey.1}" },
+          "padding-inline": { "value": "24px" },
```

Comment:
Hiervoor gebruiken wij de space.tokens.json.
In dit geval kun je dit doen "padding-inline": { "value": "{denhaag.space.inline.xl}" }